### DISCUSSION TITLE
Password session

### DISCUSSION THREAD

**Discussion by Maxichoucroutte** [bitfocus/companion#2713] [General]
Hi !
After installing the software with the terminal on Debian, I rebooted to go to the "Companion" session but it ask me a password that I didn't found on the doc. Also the terminal didn't asked me to create one or it didn't show me what kind of password it could be during the installation.

Can you give me this info ?

Thanks !

**Comment by JeLuF** [comment]
How did you install the sofware? E.g. did you use the download from bitfocus.io and `tar zxvf`, or the source code from github? 

How did you start it after a reboot?

What do you mean by 'go to the "Companion" session'? Did you go to http://_your-debian-box_:8000/ ?

**Comment by Maxichoucroutte** [comment]
Hi JeLuf,
Thank you for your answer.

So to explain my problem, I have to give you some details.
I work on video (stream, Multicam, etc.) and I just bought some monitors, a BlackMagic VideoHub 20x20 and a Smart Control Pro as remote for the 20x20. After some uses, I noticed that it was uneasy to use the Smart Control to chose the camera I want to control. So I want to use a stream deck for it. My idea was to use a RPi with just a mouse + keyboard + screen instead of using another computer (I almost have plenty of them on my table for the PPT, the stream, the blackmagic software control, etc.

I'm not alone when I work and everybody need a different computer for their jobs

And there we are... I'm on a RPi with Raspbian and I want to use Companion as I do on my Mac or my PC. But when I follow the instructions from the README note, I finally have a new session called "Companion" which ask me a password to have access to the session. I didn't find anything about it...

![IMG_7315](https://github.com/bitfocus/companion-module-generic-http/assets/99727423/28b0f89b-f077-4c29-942a-6242960cdfa3)

**Comment by Julusian** [comment]
> I finally have a new session called "Companion" which ask me a password to have access to the session. 

You can't login as the companion user, you should continue to use the normal pi user or whatever other one you setup.  
That account exists so that companion can run as its own user and have the security benefits that provides. It doesn't have a password, and logging in as it is disabled.

I assume you used the companion-pi install script, so once you have logged in as another user go to `http://localhost:8000` in your browser and it should load the companion interface

**Reply by Maxichoucroutte** [reply]
Hi
I want to apologize for my late answer and to thank you for the help. Finally, I bought a RPi 5 and it solved my problems. I think my other RPi was to tired and note in a good condition...

So I downloaded the folder from the Bitfocus Companion website and simply double-clic on the companion-launcher file then it works.

Thanks a lot again.
